packages/app/src/sections/futures/Trades/Trades.tsx

```diff
@@ -67,80 +71,220 @@ const Trades = memo(() => {
 				},
 				{
 					header: () => <TableHeader>{t('futures.market.user.trades.table.date')}</TableHeader>,
 					accessorKey: 'time',
-					cell: (cellProps) => (
-						<GridDivCenteredRow>
-							<TimeDisplay value={cellProps.getValue()} />
-						</GridDivCenteredRow>
+					cell: (cellProps) => <TimeDisplay value={cellProps.getValue()} />,
+					enableSorting: true,
+				},
+				{
+					header: () => (
+						<TableHeader>{t('futures.market.user.trades.table.price-type')}</TableHeader>
 					),
-					size: 90,
+					accessorKey: 'value',
+					sortingFn: 'basic',
+					cell: (cellProps) => {
+						return (
+							<FlexDivCol>
+								<Currency.Price price={cellProps.getValue()} />
+								<Body color="secondary">{cellProps.row.original.type}</Body>
+							</FlexDivCol>
+						)
+					},
 					enableSorting: true,
 				},
 				{
-					header: () => <TableHeader>{t('futures.market.user.trades.table.side')}</TableHeader>,
-					accessorKey: 'side',
+					header: () => <TableHeader>{t('futures.market.user.trades.table.size')}</TableHeader>,
+					accessorKey: 'amount',
 					sortingFn: 'basic',
-					cell: (cellProps) => <PositionType side={cellProps.getValue()} />,
-					size: 60,
+					cell: (cellProps) => {
+						return (
+							<FlexDivCol>
+								<Currency.Price
+									price={cellProps.getValue()}
+									currencyKey={cellProps.row.original.displayAsset!}
+									showCurrencyKey
+								/>
+								<Currency.Price
+									price={cellProps.row.original.notionalValue}
+									formatOptions={{ truncateOver: 1e6 }}
+									colorType="secondary"
+								/>
+							</FlexDivCol>
+						)
+					},
 					enableSorting: true,
 				},
 				{
-					header: () => <TableHeader>{t('futures.market.user.trades.table.price')}</TableHeader>,
-					accessorKey: 'value',
+					header: () => <TableHeader>{t('futures.market.user.trades.table.pnl')}</TableHeader>,
+					accessorKey: 'netPnl',
 					sortingFn: 'basic',
 					cell: (cellProps) => {
-						return <>{formatDollars(cellProps.getValue(), { suggestDecimals: true })}</>
+						return cellProps.getValue().eq(0) ? (
+							'--'
+						) : (
+							<FlexDivRowCentered columnGap="5px">
+								<ColoredPrice priceChange={cellProps.getValue().gt(0) ? 'up' : 'down'}>
+									{formatDollars(cellProps.getValue(), { maxDecimals: 2 })}
+								</ColoredPrice>
+								<HelpIcon />
+							</FlexDivRowCentered>
+						)
 					},
-					size: 90,
 					enableSorting: true,
 				},
 				{
 					header: () => (
-						<TableHeader>{t('futures.market.user.trades.table.trade-size')}</TableHeader>
+						<TableHeader>{t('futures.market.user.trades.table.fees-funding')}</TableHeader>
 					),
-					accessorKey: 'amount',
 					sortingFn: 'basic',
-					cell: (cellProps) => (
-						<>{formatCryptoCurrency(cellProps.getValue(), { suggestDecimals: true })}</>
+					accessorKey: 'feesPaid',
+					cell: (cellProps) => {
+						return (
+							<FlexDivCol>
+								{cellProps.getValue().eq(0) ? (
```

Comment:
Not sure I understand this conditional, what are the differences in display when the value is 0?

Reply:
That's true. Let's keep it simple. Updated.